sources/packages/api/src/testHelpers/controllers/auth-test/auth-test.controller.ts

```diff
@@ -51,5 +58,34 @@ export class AuthTestController {
    */
   @Roles()
   @Get("/authenticated-route-by-non-existing-role")
-  async authenticatedRouteByNonExistingRole(): Promise<void> {}
+  // eslint-disable-next-line @typescript-eslint/no-empty-function
+  authenticatedRouteByNonExistingRole(): void {
+    // Intentionally blank. The goal is to test the
+    // decorator and the HTTP response only.
+  }
+
+  /**
+   * Only authenticated users with specific group will have access to this endpoint.
```

Comment:
method does not have `* @param userToken`

Reply:
I will remove it!

Reply:
Removed!